updating/index.adoc

```diff
@@ -0,0 +1,64 @@
+* xref:../updating/updating-restricted-network-cluster.adoc#update-service-uninstall[Uninstalling the OpenShift Update Service Operator]
+
+['updating-clusters-overview-upgrade-channels-and-releases']
+== Upgrading channels and releases
+xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Upgrading channels and releases]: Upgrade channels allow you to choose an upgrade strategy. Upgrade channels are connected to a minor version of {product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install; the openshift-install binary file for a specific version of {product-title} always installs that version. If you do not want the Cluster Version Operator to fetch available updates from the {product-title} Update Service (OSUS), you can use the `oc adm upgrade channel` command in the `oc` to configure an empty channel. For more information, see the following information:
+
+* xref:../updating/understanding-upgrade-channels-release.adoc#upgrade-version-paths_understanding-upgrade-channels-releases[Upgrading version paths]
+* xref:../updating/understanding-upgrade-channels-release.adoc#fast-stable-channel-strategies_understanding-upgrade-channels-releases[Understanding Fast and stable channel use and strategies]
+* xref:../updating/understanding-upgrade-channels-release.adoc#restricted-network-clusters_understanding-upgrade-channels-releases[Understanding restricted network clusters]
+* xref:../updating/understanding-upgrade-channels-release.adoc#switching-between-channels_understanding-upgrade-channels-releases[Switching between channels]
+
+['updating-clusters-overview-update-cluster-using-web-console']
+== Updating a cluster within a minor version using the web console
+xref:../updating/updating-cluster-within-minor.adoc#updating-cluster-within-minor[Updating a cluster within a minor version using the web console]: You can update an {product-title} cluster by using the web console. You can the update a cluster between minor versions. For more information, see the following items:
+
+* xref:../updating/updating-cluster-within-minor.adoc#update-upgrading-web_updating-cluster-within-minor[Updating a cluster by using the web console]
+* xref:../updating/updating-cluster-within-minor.adoc#update-changing-update-server-web_updating-cluster-within-minor[Changing the update server by using the web console]
+
+['updating-clusters-overview-update-cluster-using-cli']
+== Updating a cluster within a minor version using the command-line interface (CLI)
+xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version using the CLI]: You can update an {product-title} cluster by using the `oc`. You can the update a cluster between minor versions. For more information, see the following actions:
```

Comment:
The same issue with https://github.com/openshift/openshift-docs/pull/42047/files#r847959360